test/unit/helpers/xhr/MFA_ENROLL_totp_success.js

```diff
@@ -32,7 +32,7 @@ define({
             "factorResult": "WAITING",
             "_links": {
               "qrcode": {
-                "href": "../../../test/assets/1x1.gif",
+                "href": "../../../test/unit/assets/1x1.gif",
```

Comment:
nit: We use `1x1.gif` here, but the file is actually `1x1.GIF`. The new file (`1x1v2.gif`) uses lowercase, so maybe we should change the name of `1x1.GIF` to lowercase while we're at it.